timemachine/cpp/src/nonbonded_interaction_group.hpp

```diff
@@ -69,13 +70,14 @@ template <typename RealType> class NonbondedInteractionGroup : public Potential
     void set_atom_idxs_device(
         const int NC, const int NR, unsigned int *d_column_idxs, unsigned int *d_row_idxs, const cudaStream_t stream);
 
-    void set_atom_idxs(const std::vector<int> &atom_idxs);
+    void set_atom_idxs(const std::vector<int> &row_atom_idxs, const std::optional<std::set<int>> &col_atom_idxs);
```

Comment:
Should this file `#include <optional>`? (I get compilation errors locally without this)